src/test/java/com/emc/mongoose/storage/driver/kafka/integration/CreateRecordTest.java

```diff
@@ -34,29 +35,29 @@ public void setup() {
     Properties producer_properties = new Properties();
     producer_properties.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, host_port);
     kafkaProducer =
-      new KafkaProducer<>(producer_properties, new StringSerializer(), new ByteArraySerializer());
+        new KafkaProducer<>(producer_properties, new StringSerializer(), new ByteArraySerializer());
     adminClient = KafkaAdminClient.create(properties);
   }
 
   @Test
   public void testCreateRecord() throws Exception {
     adminClient.createTopics(Collections.singletonList(new NewTopic(TOPIC_NAME, 1, (short) 1)));
-    final byte[] one_mb_of_data = new byte[999970];
+    final byte[] one_mb_of_data = new byte[900000];
```

Comment:
1. What is the purpose of this change?
2. Camel case naming should be used instead of the snake case for any java code.
3. The variable name ("1MB") seem to not represent the actual variable value :)

Reply:
Value 999970 is too large to accept by kafka server and test fails with exception RecordTooLarge.